packages/fxa-settings/src/components/LinkedAccounts/en-US.ftl

```diff
@@ -0,0 +1,8 @@
+## Linked Accounts section
+
+la-heading = Linked Accounts
+la-description = You have authorized access to the following accounts.
+la-unlink-button = Unlink
+la-unlink-account-button = Unlink
+la-unlink-heading = Unlink from third party account
+la-unlink-content = Are you sure you want to unlink your Google account? Unlinking your account does not automatically sign you out of those services, to do that you will need to manually sign out from Connected services section.
```

Comment:
Minor improvement to wording?

_to do that you will need to manually sing out from **the** Connected **S**ervices section._